packages/core/src/ceramic.ts

```diff
@@ -585,6 +586,20 @@ class Ceramic implements CeramicApi {
     return this._networkOptions.supportedChains
   }
 
+  /**
+   * Load all the pinned documents, re-request PENDING or PROCESSING anchors.
+   */
+  async recoverDocuments() {
```

Comment:
Shouldn't this method be called on startup?

Reply:
It is called, in cli/daemon. Current tests apply a lot of mocks, that (a) are too daunting to change for this little thing, (b) will be heavily redone wrt mocking while state refactoring.

Reply:
But what if I start a ceramic core instance programmatically? Seems like it should be called when you create a ceramic instance?